src/components/ErrorBoundary/index.tsx

```diff
@@ -0,0 +1,22 @@
+import BaseErrorBoundary, {LogError, BaseErrorBoundaryProps} from './BaseErrorBoundary';
+import Log from '../../libs/Log';
+
+const logError: LogError = (errorMessage, error, errorInfo) => {
+    // Log the error to the server
+    Log.alert(`${errorMessage} - ${error.message}`, {errorInfo}, false);
+};
+
+function ErrorBoundary({errorMessage, children}: BaseErrorBoundaryProps) {
```

Comment:
Before this migration `logError` was a default value and it was possible to override `logError` in props.

Let's make `logError` a default value/or imo a better option:
```suggestion
function ErrorBoundary({errorMessage, children}: Omit<BaseErrorBoundaryProps, 'logError'>) {
```

Reply:
updated